engines/python/src/main/java/ai/djl/python/engine/PyEnv.java

```diff
@@ -57,17 +58,16 @@ public class PyEnv {
 
     private boolean failOnInitialize = true;
 
+    private boolean enableVenv;
+    private boolean venvCreated;
+
     /**
      * Constructs a new {@code PyEnv} instance.
      *
      * @param mpiMode true to use MPI launcher
      */
     public PyEnv(boolean mpiMode) {
         this.mpiMode = mpiMode;
-        pythonExecutable = Utils.getenv("PYTHON_EXECUTABLE");
```

Comment:
With your implementation, no need to lazy get

Reply:
Changed.